src/sentry/demo/data_population.py

```diff
@@ -1130,10 +1171,37 @@ def handle_react_python_scenario(self, react_project: Project, python_project: P
             )
         self.assign_issues()
 
+    def handle_mobile_scenario(self, ios_project: Project, android_project: Project):
+        with sentry_sdk.start_span(op="handle_mobile_scenario", description="populate_errors"):
+            self.populate_generic_error(
+                ios_project, "errors/ios/exc_bad_access.json", 3, starting_release=1
+            )
+            self.populate_generic_error(
+                ios_project, "errors/ios/handled.json", 4, starting_release=2
+            )
+            self.populate_generic_transaction(
+                ios_project, "transactions/ios/ios_transaction.json", 2
+            )
+            self.populate_generic_error(
+                android_project, "errors/android/out_of_bounds.json", 5, starting_release=1
+            )
+            self.populate_generic_error(
+                android_project, "errors/android/app_not_responding.json", 2, starting_release=2
+            )
+        self.assign_issues()
+
 
 def handle_react_python_scenario(react_project: Project, python_project: Project, quick=False):
     """
     Handles all data population for the React + Python scenario
     """
     data_population = DataPopulation(python_project.organization, quick)
     data_population.handle_react_python_scenario(react_project, python_project)
+
+
+def handle_mobile_scenario(ios_project: Project, android_project: Project, quick=False):
+    """
+    Handles all data population for the iOS + Android + React-Native scenario
+    """
+    data_population = DataPopulation(ios_project.organization, quick)
```

Comment:
@roggenkemper we can move the instantiation of `DataPopulation` to a higher level so we can keep the `generate_releases` method as a class method.

Reply:
moved to create_demo_org